clcache.py

```diff
@@ -29,8 +29,13 @@
 try:
     import scandir # pylint: disable=wrong-import-position
     WALK = scandir.walk
+    LIST = scandir.scandir
 except ImportError:
     WALK = os.walk
+    try:
+        LIST = os.scandir
+    except:
+        LIST = os.listdir
```

Comment:
When would `os.scandir` not be available? Note that our minimum Python version is 3.3.